contracts/Comptroller.sol

```diff
@@ -1084,6 +1129,17 @@ contract Comptroller is ComptrollerV3Storage, ComptrollerInterface, ComptrollerE
     function _become(Unitroller unitroller) public {
         require(msg.sender == unitroller.admin(), "only unitroller admin can change brains");
         require(unitroller._acceptImplementation() == 0, "change not authorized");
+
+        // Intially, all markets have unlimited borrowing
```

Comment:
I think it's safer to use `0` as the "unlimited" feature (since we still have a pause function anyway) and thus we don't need to do this constructor at all.